### PR TITLE
Bug fix for range support reordering

### PR DESCRIPTION
**Pull request by HanJiyao**
This PR correct the range command that support reorder `from/` `to/`

### CODE REVIEW

File: src/main/java/seedu/contax/logic/commands/RangeCommand.java [hunk 1]
```diff
@@ -34,6 +37,8 @@
             + PREFIX_PHONE + "12345678 "
             + PREFIX_ADDRESS + "new address ";
 
+    private final Logger logger = LogsCenter.getLogger(LogicManager.class);
```
Comment (sharpstorm) [inline src/main/java/seedu/contax/logic/commands/RangeCommand.java:40]: What is the purpose of this logging? You are introducing a cyclic dependency and I do not see any other command classes doing it.

**Review by sharpstorm — CHANGES_REQUESTED** [review]
Kindly explain the purpose of logging, it looks unnecessary.

File: src/main/java/seedu/contax/logic/commands/RangeCommand.java (hunk 1 above)
Comment (HanJiyao) [reply src/main/java/seedu/contax/logic/commands/RangeCommand.java:40]: The original logger seems only record entire command, logging the individual new command may help in debuging or something

Comment (HanJiyao) [reply src/main/java/seedu/contax/logic/commands/RangeCommand.java:40]: Ohh the cyclic dependency I change the class later

**Review by sharpstorm — APPROVED** [review]
LGTM.

**Review by dandaandaaaaaan — APPROVED** [review]
lgtm thanks

**Pull request merged**